tests/test_batch.py

```diff
@@ -63,7 +64,7 @@ def encode_response(self, output) -> Response:
         return {"output": float(output)}
 
 
-class SimpleLitAPI2(LitAPI):
+class SimpleTorchAPI(LitAPI):
```

Comment:
Maybe I'm missing a point, but since it's a class inherited from `LitAPI` it should not be called `SimpleTorchAPI`.
As I understand the class above is a batch version of this class, then maybe this one should be `SimpleLitAPI` and the one above should be `class SimpleBatchLitAPI(SimpleLitAPI)`?

Reply:
valid point @Andrei-Aksionov, the reason I did't use the suggested names is because we already have `LitServe.examples.SimpleLitAPI` and `SimpleBatchLitAPI` for a different purpose.